src/precice/tests/ParallelTests.cpp

```diff
@@ -915,7 +915,7 @@ void runTestAccessReceivedMesh(const std::string         configName,
       std::vector<double> ownCoordinates(ownMeshSize * dim);
       interface.getMeshVerticesAndIDs(meshID, ownMeshSize, ownIDs.data(), ownCoordinates.data());
       BOOST_TEST(ownIDs == ids);
-      BOOST_TEST(ownCoordinates == ownCoordinates);
+      BOOST_TEST(testing::equals(ownCoordinates, ownCoordinates)); // @todo looks wrong
```

Comment:
@DavidSCN or @fsimonis can you comment on this line? I will merge the PR now, because this line is not related, but we should remove or fix it.

Reply:
Yes,this looks wrong :wink:

Reply:
Ok. Any Idea how to make this less wrong? :smirk:

Reply:
My bad, I will fix it

Reply:
Done